jarjar-gradle/src/main/java/org/anarres/gradle/plugin/jarjar/JarjarPlugin.java

```diff
@@ -10,8 +10,6 @@
 import org.gradle.api.Project;
 import org.gradle.api.file.FileCollection;
 import org.gradle.api.internal.ClosureBackedAction;
-import static org.bouncycastle.asn1.x500.style.RFC4519Style.c;
-import static org.gradle.internal.Transformers.name;
```

Comment:
I, too, had to remove these two imports for `gradle jar` to finish without build errors. Might be worth making a minimal PR to just fix this, if it has some chance of getting merged...